packages/components/message-box/src/messageBox.ts

```diff
@@ -162,7 +165,7 @@ const MESSAGE_BOX_DEFAULT_OPTS: Record<
 }
 
 MESSAGE_BOX_VARIANTS.forEach((boxType) => {
-  MessageBox[boxType] = messageBoxFactory(boxType)
+  ;(MessageBox as Record<string, any>)[boxType] = messageBoxFactory(boxType)
```

Comment:
Why is this not consistent with line 216?

Reply:
This line has type error， I change this in type，Wait， I will merge the latest code